editor/test/integration/gui_test.clj

```diff
@@ -786,10 +822,7 @@
   (#'gui/move-child-node! node-id offset))
 
 (defn- scene-gui-node-map [scene]
-  (into {}
-        (map (juxt :label :node-id)
-             (tree-seq :children :children (g/node-value scene :node-outline)))))
-
+  (g/node-value scene :node-ids))
```

Comment:
The `node-ids` output on the `GuiSceneNode` is exactly this - a map of `{:name :node-id}`. Although `:name` is called `:id` on the `GuiNode`.